apps/librelingo_types/librelingo_types/data_types.py

```diff
@@ -134,7 +135,8 @@ class Course(
         ),
         dictionary=[dict_item1, dict_item2, dict_item3, dict_item4],
         repository_url="https://example.com",
-        setttings=Settings()
```

Comment:
wow, what a typo that has been. it's interesting that it never caused any issues